minecraft_servers/crystopia/manifest.json

```diff
@@ -0,0 +1,28 @@
+{
+  "server_name": "crystopia",
+  "nice_name": "Crystopia",
+  "direct_ip": "play.crystopia.net",
+  "server_wildcards": [
+    "mc.crystopia.net"
+  ],
+  "social": {
+    "web": "https://crystopia.net",
+    "web_shop": "none",
```

Comment:
what?! 😂 I would understand null or an empty string - But where did you get the information that we can and would handle "none" as string? 😂
So if you do not have a web shop, remove the key. It is not required. And the user will not gets a redirect to "none"!